docs/root/configuration/overview/v2_overview.rst

```diff
@@ -499,10 +501,35 @@ to
 with the effect that the LDS stream will be directed to *some_ads_cluster* over
 the shared ADS channel.
 
+.. _config_overview_v2_status:
+
 Status
 ------
 
-The current API status is tracked `here
-<https://github.com/envoyproxy/data-plane-api#status>`_. All features described
-in the :ref:`v2 API reference <envoy_api_reference>` are implemented unless
-otherwise noted.
+All features described in the :ref:`v2 API reference <envoy_api_reference>` are
+implemented unless otherwise noted. In the v2 API reference and the
+`v2 API repository
+<https://github.com/envoyproxy/data-plane-api/tree/master/api>`_, all protos are
+*frozen* unless they are tagged as *draft* or *experimental*. Here, *frozen*
+means that we will not break wire format compatibility.
+
+*Frozen* protos may be further extended, e.g. by adding new fields, in a
+manner that does not break `backwards compatibility
+<https://developers.google.com/protocol-buffers/docs/overview#how-do-they-work>`_.
+Fields in the above protos may be later deprecated, subject to the
+`breaking change policy
+<https://github.com/envoyproxy/envoy/blob/master//CONTRIBUTING.md#breaking-change-policy>`_,
+when their related functionality is no longer required. While frozen APIs
+have their wire format compatibility preserved, we reserve the right to change
+proto namespaces, file locations and nesting relationships, which may cause
```

Comment:
FWIW, I'm a little concerned that we have not namespaced the protos enough as it is. I.e., should all RDS messages be in an RDS namespace, etc.? Should each filter be in its own namespace? I think this is going to bite us eventually. Is it worth it to just fix this now and get it over with before there are too many users?

Reply:
Yes, I agree, as the APIs have grown it's now evident that we could benefit from some namespace isolation. I think we should rip the band aid and do it before the 1.5 merge. Created https://github.com/envoyproxy/data-plane-api/issues/270 to track.